mautic/mautic
# Removed uses of manually setting content-length headers

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | Y
| New feature?  | N 
| BC breaks?    |  N
| Deprecations? | N
| Fixed issues  |  

## Description

In some cases where apache is behind a load balancer or even in the case of apache+mod_fcgi, manually setting the content-length headers resulted in truncated content. It also prevented support for http 1.1 chunking. 

## Steps to reproduce the bug (if applicable)

Difficult without the right web server setup. 

## Steps to test this PR

Mainly ensure that the following functions still work:

1. Adding campaign events
2. Adding form fields and actions
3. Dashboard adding/editing widgets
4. Dashboard export of layout
5. Adding/editing lead notes
6. Adding point trigger events
